src/styles/block/section.scss

```diff
@@ -0,0 +1,104 @@
+.main {
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.